src/components/UserWrapper/UserWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import {blockNew} from '../utils/cn';
+
+import type {UserWrapperProps, UserWrapperSize} from './types';
+
+import './UserWrapper.scss';
+
+const SMALL_SIZES: Set<UserWrapperSize> = new Set(['3xs', '2xs', 'xs']);
+
+const b = blockNew('user-wrapper');
+
+export const UserWrapper = React.forwardRef<HTMLDivElement, UserWrapperProps>(
```

Comment:
I have doubts about this component, looks a bit ugly to have User and UserWrapper. Does the only purpose of the component is to provide custom avatars?